integration-cli/test_vars_windows.go

```diff
@@ -0,0 +1,8 @@
+// +build windows
+
+package main
+
+const (
+	// idetifies if test suite is running on a unix platform
+	isUnixCli = false
```

Comment:
Also at some point I can see us running against windows daemon, so isUnix would probably get refactored to this anyway to clarify what it actually refers to.